lib/standards/aria-roles.js

```diff
@@ -301,7 +302,7 @@ const ariaRoles = {
     superclassRole: ['section']
   },
   listbox: {
-    type: 'widget',
+    type: 'composite',
```

Comment:
```suggestion
    type: 'widget',
```

Reply:
Add comment above `type` with `// Note: spec difference` as we want to keep `widget` role